### DISCUSSION TITLE
How to verify the created token?

### DISCUSSION THREAD

**Discussion by ShahadatAnik** [Mister-Hope/bcrypt-ts#498] [Q&A]
i am  creating token using await signJWT for api and sending to the client. Now i  want to verify the given token that i have created? How to do that in this library. I haven't found any funtion like that.

```
const payload = {
    uuid: value.uuid,
    username: value.name,
    email: value.email,
    can_access: value.can_access,
    exp: Math.floor(Date.now() / 1000) + 60 * 60 * 24,
  };

  const token = await signJWT(payload, 'MY_PRIVATE_KEY');
```

**Comment by Mister-Hope** [comment]
What do you mean by "how to verify a token"?

The lib is providing a way to hash any string with random salt, and then compare it with the original one, this should be clear on README.

**Comment by ShahadatAnik** [comment]
but, how to verify that, this my generated token?
